Source/story.ni

```diff
@@ -647,16 +651,17 @@ Before printing the name of something audible while listening to a room:
 	say "[sound] from the ";
 													[---------------------------]
 													
-The newspaper, paper is a thing. The newspaper, paper is carried by the Boy.  The description of the newspaper is "[If the Boy is carrying the newspaper]The boy pulls the newspaper out of your face, saying 'Oi, you can't look before ya buy![else]The paper is dated October 4, 1872. Seems to be a high end newspaper, because you can't understand many of the words. There is, however, one section that catches your eye.
+The newspaper, paper is a thing. The newspaper, paper is carried by the Boy.  The description of the newspaper is "[If the Boy is carrying the newspaper]The boy pulls the newspaper out of your face, saying 'Oi, no peeking before ya buy![else]The paper is dated October 4, 1872. Looks like a gentleman's paper - there are not many words you can understand. There is, however, one headline that catches your eye.
 			
 'Cotton Gin Accident leaves Factory Worker Dead'
 			
-The paper then explained what had happened to Jones' wife. She got too close to the machine, the paper said, and before she knew it, it sucked her in and tore her apart. Only some of her remains were found.
+It details a tragic incident that happened to a worker you used to know: Cathay - Jones' wife.  From the grisly remains of Cathay surrounding a machine, investigators concluded that she accidentally wandered too close to the machine and was swallowed up by its turning mechanism. Likely, and thankfully, it was quick. 
```

Comment:
wandered to close to the machine --> wandered too close
Likely, and thankfully, it was quick --> Thankfully, it was most likely a quick death
Not sure about the second revision